resume.md

```diff
@@ -1,49 +1,48 @@
-# FIRST LAST NAME
+# JOSHUA FAJARDO
 
-(123) 456 - 7890 | email@gmail.com
+(123) 456 - 789| email@gmail.com
 
-www.website.com | github.com/myusername | linkedin.com/myusername
+joshuafajardo.com | github.com/joshuafajardo | linkedin.com/joshuafajardo
 
 ## EDUCATION
-**University of College** | B.S. Degree ... *MONTH 20XX*
-* **Coursework**: Data Structures, Algorithms, Upper Division CS Elective, Math Course, Statistics, Data Science, ETC.
+
+**University of California, Berkeley** | B.S. Electrical Engineering & Computer Sciences | GPA: 3.71 ... *FALL 2021*
+
+* **Coursework**: Data Structures, Algorithms, Machine Structures, Statistics & Probability, Linear Algebra, Multivariable Calculus, Optimization,
 
 ## SKILLS
-* **Languages**: Python, Java, C, SQL, ...
-* **Libraries/Frameworks**: Pandas, TensorFlow, PyTorch, ...
-* **Version Control**: Git, Bitbucket, Sourcetree, GithubDesktop
-* *...*
+
+* **Languages**: Python, Java, C, SQL, JavaScript, HTML, CSS, MATLAB, Arduino,
+* **Frameworks and Tools**: ReactJS, TCP/IP, Threading, Git, NumPy, Socket, Socket.IO, Tkinter,
 
 ## EXPERIENCE
-**Software Engineer**: Company | City, State ... *Month 20XX - Present*
-* Implemented XYZ Microservice using TECHNOLOGY to delivery PRODUCT to 123 Customers Globally
-* Architected CORE INFRASTRUCTURE LIBRARY used across X teams to BLAH BLAH
-* I did X for Y to accomplish Z
-* ...
 
-**Software Engineer Intern**: Company | City, State ... *Month 20XX - Month 20XX*
-* Conducted A/B Test using FRAMEWORK to test EXPERIENCE in order to increase METRIC resulting in NUMBER. 
-* Utilized DEBUG TECHNOLOGY to resolve recurring crashes (X% of sessions) in beta before hitting production..
-* ...
+**Undergraduate Researcher, Software/Hardware**: Embodied Dexterity Group| Berkeley, CA ... *November 2019 - Present*
+
+* Led team of 3 in designing a physics engine and GUI in Python to interface with a prosthetic arm. Incorporated the Model/View/Controller (MVC) design pattern to organize functions and methods.
+* Established TCP connection with LabVIEW program using Python's socket library to ensure the simulation data was sufficiently accurate for use in research.
+* Utilized multithreading to run communication/physics and GUI processing in parallel, allowing the GUI to refresh at a consistent framerate.
```

Comment:
why does this specific use case need to be multithreaded?  What were the challenges that you overcame